contents/meta/404.md

```diff
@@ -3,4 +3,17 @@ title: Page not found
 subtitle: Whoopsies!
 ---
 
-We are sorry, but the page you were looking for is nowhere to be found.
+### We are sorry, but the page you were looking for is nowhere to be found.
+{: class="text-center"}
+
+We are sorry.
+{: class="text-center"}
+
+![We are vewwy sowwy.][cat]{: class="mx-auto d-block img-fluid"}
+
+(*Photo: Mr Abdoun the scottish cat by [Mashhour][commons-cat], [CC BY-SA 3.0][cc-by-sa]*)
```

Comment:
How about Mr Wiggles, who is even sadder? https://github.com/diaspora/diaspora/blob/develop/app/assets/images/sadcat.jpg

Reply:
The original link in https://github.com/diaspora/diaspora/pull/3781 expired, so I no longer can validate that the image was licensed under a CC license. In addition, all other photos on that user's profile are not licensed under a CC.